Makefile

```diff
@@ -24,7 +24,7 @@ lib_exif_0 =
 lib_exif_1 = -lexif
 lib_gif_0 =
 lib_gif_1 = -lgif
-ldlibs = $(LDLIBS) -lImlib2 -lX11 -lXft -lfontconfig \
+ldlibs = $(LDLIBS) -lImlib2 -lX11 -lXft -lfontconfig -lXrender\
```

Comment:
Is this needed? I don't see any new includes or use of Xrender in general.
And if it is needed, going to object to the new mandatory dependency.

Reply:
It seems to compile and work fine without, so I do not know why the author included it. Maybe somebody else has some more information?

Reply:
If this is the entire patch and there are no missing files/commits, then the above diff isn't needed. It simply isn't used. The only thing it does as is to make the build failed in Xrender isn't installed. Other than this line, the change looks fine to me.

Reply:
This is the whole patch.

To keep the author's patch as-is, I'll add a commit on top of this to remove the dependency.